ignored-packages.txt

```diff
@@ -0,0 +1,2 @@
+matplotlib
+pypiwin32
```

Comment:
Same comment as for ``accepted-licenses.txt`` applies here.

Reply:
Done.